types/v56/types.go

```diff
@@ -1778,7 +1780,7 @@ type LbPoolMember struct {
 	Port        int    `xml:"port"`
 	MaxConn     int    `xml:"maxConn,omitempty"`
 	MinConn     int    `xml:"minConn,omitempty"`
-	Condition   string `xml:"condition,omitempty"`
+	Condition   string `xml:"condition"`
```

Comment:
Interesting change. Curious, why only this field got it, but others above not?

CC: @Didainius

Reply:
I found that in API diff documentatioin

Reply:
Interested in a bit more details. This is NSX-V API so v27 -> v29 shouldn't impact it. Any example how to replicate this problem?

Reply:
Run many times. At least on my env. Seems its just unordered list - so we can't impact behaviour

Reply:
Did it cause any issues? Because this is a proxied API I think it should depend on NSX version rather than vCD. For example the recent NSX version 6.4 states that this field is optional.

<img width="983" alt="image" src="https://user-images.githubusercontent.com/15804230/68830377-2aca4380-06b4-11ea-94c6-fe83521d1207.png">

https://docs.vmware.com/en/VMware-NSX-Data-Center-for-vSphere/6.4/nsx_64_api.pdf (page 415)

Our provider codebase usually sends this parameter anyway, but having it without `omitempty` may send an empty value for someone who is not setting it and get an error so I'd say we could leave as it was unless the behavior of testing has changed with v29.

Reply:
Revert back. This is NSX-V type not vCD API.